### ISSUE TITLE
Incorrect handling of comment starting with `//!`

### ISSUE THREAD

**Issue by jpsacha** [bytedeco/javacpp#32]
When C++ has a comment starting with `//!`, like:
```cpp
//! Interface to instant camera parameters
class CInstantCameraParams_Params
{ ...
```

The generated Java code is missing initial `/` in the comment, so we get:
```java
 *  Interface to instant camera parameters */
@Namespace("Basler_InstantCameraParams") @NoOffset public static class CInstantCameraParams_Params extends Pointer {
```
And the code does not compile.

Notice that there is space-star (` *`) instead of slash-star (`/*`).

**Comment by saudet** [comment]
I get the correct `/**`. It might have something to do with the context... Could you post the line before as well?

**Comment by jpsacha** [comment]
Here is all the way from the beginning of the file:

```cpp




//-----------------------------------------------------------------------------
//  (c) 2004-2008 by Basler Vision Technologies
//  Section: Vision Components
//  Project: GenApi
//-----------------------------------------------------------------------------
/*!
\file
\brief Interface to instant camera parameters
  */

  //-----------------------------------------------------------------------------
  //  This file is generated automatically
  //  Do not modify!
  //-----------------------------------------------------------------------------

  #ifdef __GNUC__
  #define GENAPI_DEPRECATED_FEATURE __attribute__((deprecated))
  #elif defined(_MSC_VER) 
  #define GENAPI_DEPRECATED_FEATURE __declspec(deprecated)
  #else
  #define GENAPI_DEPRECATED_FEATURE
  #endif


  

#ifndef Basler_InstantCameraParams_PARAMS_H
#define Basler_InstantCameraParams_PARAMS_H

#include <GenApi/IEnumerationT.h>
#include <GenApi/NodeMapRef.h>
#include <GenApi/DLLLoad.h>

//! The namespace containing the device's control interface and related enumeration types
namespace Basler_InstantCameraParams
{

    //**************************************************************************************************
    // Enumerations
    //**************************************************************************************************
    

    //**************************************************************************************************
    // Parameter class
    //**************************************************************************************************
    

    //! Interface to instant camera parameters
    class CInstantCameraParams_Params
    {
    //----------------------------------------------------------------------------------------------------------------
    // Implementation
    //----------------------------------------------------------------------------------------------------------------
    protected:
		// If you want to show the following methods in the help file
		// add the string HIDE_CLASS_METHODS to the ENABLED_SECTIONS tag in the doxygen file
        //! \cond HIDE_CLASS_METHODS
        
            //! Constructor
            CInstantCameraParams_Params(void);

            //! Destructor
            ~CInstantCameraParams_Params(void);

            //! Initializes the references
            void _Initialize(GenApi::INodeMap*);

            //! Return the vendor of the camera
            const char* _GetVendorName(void);

            //! Returns the camera model name
            const char* _GetModelName(void);
        
        //! \endcond

    //----------------------------------------------------------------------------------------------------------------
    // References to features
    //----------------------------------------------------------------------------------------------------------------
    public:
        
	//! \name Root - Instant camera parameters.
    //@{
	/*!	
		\brief The maximum number of buffers that are allocated and used for grabbing.

	
	
		\b Visibility = Beginner
        
	
    */
    GenApi::IInteger &MaxNumBuffer;
    
    //@}
    

	//! \name Root - Instant camera parameters.
    //@{
	/*!	
		\brief The maximum number of buffers that are queued in the stream grabber input queue.

	
	
		\b Visibility = Beginner
        
	
    */
    GenApi::IInteger &MaxNumQueuedBuffer;
    
    //@}
...
```

This turns into generated code:

```java
// Parsed from <pylon/_InstantCameraParams.h>


    




//-----------------------------------------------------------------------------
//  (c) 2004-2008 by Basler Vision Technologies
//  Section: Vision Components
//  Project: GenApi
//-----------------------------------------------------------------------------
/**
\file
\brief Interface to instant camera parameters
  */

  //-----------------------------------------------------------------------------
  //  This file is generated automatically
  //  Do not modify!
  //-----------------------------------------------------------------------------

//   #ifdef __GNUC__
  public static native @MemberGetter int GENAPI_DEPRECATED_FEATURE();
  public static final int GENAPI_DEPRECATED_FEATURE = GENAPI_DEPRECATED_FEATURE();
//   #elif defined(_MSC_VER)
//   #else
//   #define GENAPI_DEPRECATED_FEATURE
//   #endif


  

// #ifndef Basler_InstantCameraParams_PARAMS_H
// #define Basler_InstantCameraParams_PARAMS_H

// #include <GenApi/IEnumerationT.h>
// #include <GenApi/NodeMapRef.h>
// #include <GenApi/DLLLoad.h>

/** The namespace containing the device's control interface and related enumeration types */

    //**************************************************************************************************
    // Enumerations
    //**************************************************************************************************
    

    //**************************************************************************************************
    // Parameter class
    //**************************************************************************************************
    

     *  Interface to instant camera parameters */
    @Namespace("Basler_InstantCameraParams") @NoOffset public static class CInstantCameraParams_Params extends Pointer {
        static { Loader.load(); }
        /** Empty constructor. */
        public CInstantCameraParams_Params() { }
        /** Pointer cast constructor. Invokes {@link Pointer#Pointer(Pointer)}. */
        public CInstantCameraParams_Params(Pointer p) { super(p); }
    
        
	/** \name Root - Instant camera parameters. */
    //@{
	/**	
		\brief The maximum number of buffers that are allocated and used for grabbing.

	
	
		\b Visibility = Beginner
        
	
    */
    public native @ByRef IInteger MaxNumBuffer(); public native CInstantCameraParams_Params MaxNumBuffer(IInteger MaxNumBuffer);
    
    //@}

```

**Comment by jpsacha** [comment]
The recent fix [b94b057](https://github.com/bytedeco/javacpp/commit/b94b057d702fc3d579589cbfa33cc5adf3ec2ba4) resolved the issue. Thanks.
